pkg/pillar/types/sriovtypes.go

```diff
@@ -0,0 +1,181 @@
+// Copyright (c) 2022 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package types
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"net"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+
+	uuid "github.com/satori/go.uuid"
+	"github.com/vishvananda/netlink"
+)
+
+const (
+	NicLinuxPath     = "/sys/class/net/"
+	NumvfsDevicePath = "/device/sriov_numvfs"
+	VfCountFieldName = "sriov-vf-count"
+)
+
+// Must match fields of EthVF in devcommon.proto
+type EthVF struct {
+	Index        uint8
+	PciLong      string // BFD notation
+	Iface        string
+	UsedByUUID   uuid.UUID
```

Comment:
I don't see any new code setting or using this UsedByUUID

Reply:
removed